### PR TITLE
Hbv3 smaller size addition

### PR DESCRIPTION
**Pull request by rafsalas19**
1. Added the following SKU sizes:
  - hb120-96rs_v3
  - hb120-64rs_v3
  - hb120-32rs_v3
  - hb120-16rs_v3

2. Modified run script, and stream test to support the SKUs

3. Modify install file to just install cuda perf test as it works for both gdr and non gdr ib write tests. (issue 46)

5.  Modified GDR IB write test to use bidirectional. It seems to increases the throughput value
  - Modified Ndv4 conf files (issue 46)

### CODE REVIEW

**Review by rajvi — APPROVED** [review]

File: customTests/azure_ib_write_bw_gdr.nhc [hunk 1]
```diff
@@ -22,7 +22,7 @@ function run_ib_bw_gdr(){
    MESSAGE_SIZE=8388608
 
    IB_WRITE_BW=`basename $IB_WRITE_BW_EXE_PATH`
-   IB_WRITE_BW_ARGS="-p $PORTARG -s $MESSAGE_SIZE -x 0 -F --report_gbits  --perform_warm_up"
+   IB_WRITE_BW_ARGS="-p $PORTARG -s $MESSAGE_SIZE -x 0 -F -b --report_gbits  --perform_warm_up"
```
Comment (jeseszhang1010) [inline customTests/azure_ib_write_bw_gdr.nhc:25]: Any reason we changed with bi-directional?

File: customTests/azure_ib_write_bw_gdr.nhc [hunk 2]
```diff
@@ -78,3 +78,5 @@ function check_ib_bw_gdr(){
    wait  ${pid_array[@]}
    return 0
 }
+
+# check_ib_bw_gdr 170.0 nd96asr_v4
```
Comment (jeseszhang1010) [inline customTests/azure_ib_write_bw_gdr.nhc]: Do we really need it? And it seems changed to 175 in [conf/nd96asr_v4.conf](https://github.com/Azure/azurehpc-health-checks/pull/50/files#diff-b4e63cc69f103889afc329719236165bb6d26b191bf18b9e0e9087c0e6927a13)

Comment (rafsalas19) [reply customTests/azure_ib_write_bw_gdr.nhc]: Good catch it was left over from testing. Let me remove

File: customTests/azure_ib_write_bw_gdr.nhc (hunk 1 above)
Comment (rafsalas19) [reply customTests/azure_ib_write_bw_gdr.nhc:25]: This was semi related to Issue 46, bidirectional flag increases the perf results. I wanted to get as close to peak perf as possible and bidirectional flag increases the throughput.

Do you see any concerns with using bidirectional traffic on a loop back test?

File: customTests/azure_ib_write_bw_gdr.nhc (hunk 2 above)
Comment (rafsalas19) [reply customTests/azure_ib_write_bw_gdr.nhc]: addressed

File: customTests/azure_ib_write_bw_gdr.nhc (hunk 1 above)
Comment (rafsalas19) [reply customTests/azure_ib_write_bw_gdr.nhc:25]: I don't see an issue with using bidirectional with this test. So I am going to continue with it. If an issue arises we can fall back.

**Pull request merged**